tests/guestlog/guestlog.go

```diff
@@ -73,7 +73,7 @@ var _ = Describe("[sig-compute]Guest console log", decorators.SigCompute, func()
 					}
 				}
 			},
-				Entry("with AutoattachSerialConsole and LogSerialConsole", true, true, true),
```

Comment:
It seems that we start 5 VMs and test nothing at all about the VM or the guest behavior.
What is the value of these 5 end-to-end tests?

How does adding any of them makes our Conformance test suite better?

Reply:
IMO since this feature is enabled by default it's worth it to add the minimal test to our Conformance.
I agree that the other permutations can probably be a simple unit test.
I  would love to add a test that performs some checks to the logs but due to https://github.com/kubevirt/kubevirt/pull/13193 currently, this is not possible.
But still think that we should at least add one to the conformance.
Thanks

Reply:
We should work on #13193 but as Dan says this test is not even e2e test.

Reply:
IMO having this test gives us a false sense of security. It tests nothing about the feature (collecting logs from guest), only that containers start when they are asked to.

Yes, we need a test for this feature, but I'm afraid this is not it :-(

Reply:
Let's see if https://github.com/kubevirt/kubevirt/pull/13422 help

Reply:
Agreed!
I think I will close it for now, since there isn't any good candidate. or do you see one?

Reply:
I'd love to keep the code cleanup, and would certainly vote for deleting these 5 tests.